tests/xtimer_mutex_lock_timeout/main.c

```diff
@@ -114,6 +152,78 @@ static int cmd_test_xtimer_mutex_lock_timeout_long_locked(int argc,
             puts("error mutex not locked");
         }
     }
+    /* to make the test easier to read */
+    printf("\n");
+
+    return 0;
+}
+
+/**
+ * @brief   shell command to test xtimer_mutex_lock_timeout
+ *
+ * This function will create a new thread with lower prio.
```

Comment:
dot at the end of the line can be removed

Reply:
Ok